benchmark_results.md

```diff
@@ -0,0 +1,96 @@
+# Benchmark Results
+
+Following numbers were gathered targeting the [process-quarkus-example](https://github.com/kiegroup/kogito-examples/tree/stable/process-quarkus-example) on different compute environments.
+
+Mainly 2 options were tested:
+1. Kogito application deployed in a Virtual machine
+2. Kogito application deployed in a Red Hat Openshift cluster.
+
+
+#### VM Details: 🖥️
+TBD
+
+#### Config Details: ⚒️
+
+- Java version: openjdk 11.0.12 2021-07-20 LTS
+- Persistence: Mongo DB v4.4.9
+- Kogito version: 1.11.0.Final
+- Concurrence Strategy: CONST_CONCURRENT_USERS
+
+## VM deployment
```

Comment:
Shouldn't we split the report in 2 sections, one for the default configuration (e.g., to be used for official report) and one with the manual updates to boost the performance? Or do we want Ivo to make the split? (leaving the team to decide)

Reply:
keep as it is, single column is also fine